ui/perfherder/alerts/AlertTable.jsx

```diff
@@ -176,6 +169,36 @@ export default class AlertTable extends React.Component {
     return { failureStatus };
   };
 
+  selectAlertsByStatus = (selectedStatus) => {
```

Comment:
This logic feels more like the responsibility of `SelectAlertsDropdown`. Wouldn't it be better to try to move it there?
All the parent needs to know is what alerts have been selected by the dropdown & whether all have been selected or not.